### PR TITLE
Adds `config-capture` command

### LINKED ISSUE #168: Create a `capture-config` command
```
# Grab the live db.
fin pull db -y --hosting-env=live

# @TODO Need an IF statement here if there is any changes.
# This is an idea:
# var_config_status=$(fin drush config:status 2>&1 | grep "No differences")
# if [[ "$var_config_status" != "[notice] No differences between DB and sync directory." ]]
# then
#     echo -e "If statement."
# else
#     echo -e "Else statement"
# fi
# echo -e "Statement: $var_config_status"
# =================================================================

# Make a new branch.
git checkout -b feature/live-config
# @TODO Make this unique somehow

# Export and commit config.
fin drush config:export -y --commit --message="Automated configuration capture from live database"

# Push and create a PR.
git push origin
gh pr create --title "Configuration from live site." --body "Automatically generated pull request from the live site configurations."

# Delete the branch.
@TODO
```

### PR DESCRIPTION
**Pull request by thejimbirch**
Closes #168

### CODE REVIEW

File: .docksal/commands/config-capture [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+# Abort if anything fails
+set -e
+
+ENVIRONMENT=${1:-"$hostingenv"}
+
+# Sets todays datetime.
+var_date=$(date +"%Y-%m-%d-%H-%M-%S")
+
+# Install the terminus-rsync-plugin plugin.
+echo -e "Install the terminus-rsync-plugin plugin"
```
Comment (sean-e-dietrich) [inline .docksal/commands/config-capture:17]: What happens if the plugin is already installed? Does it try to install it again?

Comment (thejimbirch) [reply .docksal/commands/config-capture:17]: `[notice] pantheon-systems/terminus-rsync-plugin is already installed.`

Comment (thejimbirch) [reply .docksal/commands/config-capture:17]: It works fin as is.

The only way I could see figuring out if it is installed or not was:

```
❯ fin terminus self:plugin:list
 ----------------------- ------------------------------------------------------------ ------------------- ---------------- -----------------
  Name                    Description                                                  Installed Version   Latest Version   Compatible With
 ----------------------- ------------------------------------------------------------ ------------------- ---------------- -----------------
  terminus-rsync-plugin   Rsync - Terminus Plugin that provides a quick shortcut for   1.1.0               1.1.0            ^1|^2
                          rsync-ing files to and from a Pantheon site.
 ----------------------- ------------------------------------------------------------ ------------------- ---------------- -----------------
 ```

But I can't see an easy way to be able to parse that into an IF statement.

This is similar to the `drupal-starter-installer` project with the `gh` plugins.  If they are installed, trying to install them again seems just as fast as checking if they are installed.

Comment (thejimbirch) [reply .docksal/commands/config-capture]: Updated.

**Review by sean-e-dietrich — APPROVED** [review]

**Comment by sean-e-dietrich** [comment]
Looks good! 🎊

**Pull request merged**